### PR TITLE
fix: docs: node template user data MIME BOUNDARY

### PR DESCRIPTION
**Pull request by Kostavro**
Fixes # 

**Description**

As observed in https://github.com/aws/karpenter/issues/2098, there must be a new, empty line before `--BOUNDARY`

**How was this change tested?**
Tested without the new line and got error:
```
ERROR	controller.provisioner	launching machine, creating cloud provider instance, getting launch template configs, getting launch templates, creating launch template, parsing custom user data input unreadable user data malformed MIME header: missing colon: "--BOUNDARY"
```

With the empty line karpenter is able to provision new instances with the correct user data.

**Does this change impact docs?**
- [X] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note
NONE
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/v0.22.0/concepts/node-templates.md [hunk 1]
```diff
@@ -334,6 +334,7 @@ spec:
   userData: |
     MIME-Version: 1.0
     Content-Type: multipart/mixed; boundary="BOUNDARY"
+
```
Comment (bwagner5) [inline website/content/en/v0.22.0/concepts/node-templates.md:337]: Thanks for catching this!  Can you apply this change to `website/content/en/preview/concepts/node-templates.md` instead?  We promote the preview docs when we cut a release.

**Review by bwagner5 — COMMENTED** [review]
Thanks, just one comment to apply this change to the preview docs as well

File: website/content/en/v0.22.0/concepts/node-templates.md (hunk 1 above)
Comment (Kostavro) [reply website/content/en/v0.22.0/concepts/node-templates.md:337]: hey @bwagner5 of course, but you confused me a bit ... apply the change to `preview` **instead** or **as well**?

Comment (Kostavro) [reply website/content/en/v0.22.0/concepts/node-templates.md]: hey @tzneal took me a while as I was off for a few days, but I rebased and pushed now :)

File: website/content/en/v0.20.0/faq.md [hunk 2]
```diff
@@ -90,6 +90,27 @@ See [Node NotReady]({{< ref "./troubleshooting/#node-notready" >}}) troubleshoot
 
 Though not recommended, if you need to access Karpenter-managed nodes without AWS credentials, you can add SSH keys using AWSNodeTemplate.
 See [Custom User Data]({{< ref "./concepts/operating-systems/" >}}) for details.
+Though not recommended, if you need to access Karpenter-managed nodes without AWS credentials, you can add SSH keys to Launch Templates through [Custom User Data](https://karpenter.sh/preview/aws/user-data/).
```
Comment (bwagner5) [inline website/content/en/v0.20.0/faq.md:93]: looks like this was duplicated or copied incorrectly?

File: website/content/en/v0.21.1/concepts/node-templates.md [hunk 3]
```diff
@@ -334,7 +334,9 @@ spec:
   userData: |
     MIME-Version: 1.0
     Content-Type: multipart/mixed; boundary="BOUNDARY"
+
     --BOUNDARY
+
```
Comment (bwagner5) [inline website/content/en/v0.21.1/concepts/node-templates.md:339]: extra newline

**Review by bwagner5 — COMMENTED** [review]
sorry for the delay, a few comments.  Just updating the preview docs is sufficient since these versioned docs are a moving target (v0.23.0 is out and isn't updated in this PR).

File: website/content/en/v0.20.0/faq.md (hunk 2 above)
Comment (Kostavro) [reply website/content/en/v0.20.0/faq.md]: indeed, maybe came from some UI-done rebase

**Comment by Kostavro** [comment]
> sorry for the delay, a few comments. Just updating the preview docs is sufficient since these versioned docs are a moving target (v0.23.0 is out and isn't updated in this PR).

@bwagner5 rolled back changes to docs apart from preview.

**Review by ellistarn — DISMISSED** [review]

**Review by bwagner5 — APPROVED** [review]
LGTM! Thanks!!!

**Pull request merged**